templates/desktop/enterprise.html

```diff
@@ -1,89 +1,93 @@
-{% extends "desktop/base_desktop.html" %}
+{% extends "desktop/base_desktop-v1.html" %}
 
 {% block title %}For enterprise{% endblock %}
 {% block extra_body_class %}desktop-enterprise{% endblock %}
```

Comment:
I don't actually know about any of the `extra_body_class`es. But personally I'd just grep the Sass and see if it's mentioned. If not, go ahead and remove it.